modules/stanford_profile_styles/lib/scss/component/paragraph/gallery/index.scss

```diff
@@ -7,6 +7,19 @@
     @include margin(null null 4rem null);
   }
 
+  .su-gallery-caption {
+    @include caption;
+    @include padding(.4em null null null);
+
+    @include grid-media-between('sm', 'md') {
+      float: none;
+      text-align: unset;
+    }
+
+    float: right;
+    text-align: right;
```

Comment:
This might just be my OCD and semantics, but i prefer properties at the top of the selector group.
can you move the bottom properties above or below the `@include`s at the top

Reply:
No problem.  My inclination is to have the `@include`s at the top. 😆

Reply:
Thanks. i think its just easier to read to me when i see that selector and i don't have to skip all the `grid-media-between` or child selectors. but that's just me